css/border-width/demos/different-border/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+  <title>Примеры разных вариантов рамки — background-width — Дока</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com/" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto&display=swap">
+  <style>
+    * {
+      margin: 0;
+      padding: 0;
+      font-family: "Roboto", sans-serif;
+      box-sizing: border-box;
+    }
+
+    html,
+    body {
+      height: 100%;
+    }
+
+    body {
+      background-color: #dfdfdf;
+    }
+
+    .content {
+      padding: 70px 10px 30px;
+      width: 500px;
+      max-width: 95%;
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+      margin: auto;
+    }
+
+    .text {
+      font-size: 16px;
+      padding: 10px 20px;
+      border-style: solid;
+      border-width: 3px;
+    }
+
+    .control {
+      width: 100%;
+      display: flex;
+      flex-direction: column;
+      margin-bottom: 30px;
+    }
```

Comment:
```suggestion
    .control {
      width: 100%;
      display: flex;
      flex-direction: column;
      margin-bottom: 30px;
      font-family: "Roboto Mono", monospace;
    }
```

Я бы задала моноширинный шрифт для того, что притворяется кодом. 
Главное не забыть подключить Roboto Mono =)